charts/posthog/templates/web-service.yaml

```diff
@@ -36,3 +36,42 @@ spec:
   loadBalancerSourceRanges:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+---
```

Comment:
Can you split this out into `events-service.yaml` This is getting a bit big. This is also how we do every other service in the chart (split up into their own files)